config/default.yml

```diff
@@ -316,15 +316,15 @@ RSpec/EmptyExampleGroup:
   AutoCorrect: contextual
   SafeAutoCorrect: false
   VersionAdded: '1.7'
-  VersionChanged: "<<next>>"
+  VersionChanged: '3.0'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/EmptyExampleGroup
 
 RSpec/EmptyHook:
   Description: Checks for empty before and after hooks.
   Enabled: true
   AutoCorrect: contextual
   VersionAdded: '1.39'
-  VersionChanged: "<<next>>"
+  VersionChanged: '3.0'
```

Comment:
Is this correct?

Reply:
I am not sure.

Reply:
We didn’t have VersionAdded and VersionChanged when we released v2.0.0.pre, so there’s no precedent.

Reply:
Ah, I see, those are the “contextual” cops, and right? All good then